exporters/stats_stackdriver/src/main/java/io/opencensus/exporter/stats/stackdriver/StackdriverExportUtils.java

```diff
@@ -0,0 +1,322 @@
+import io.opencensus.tags.TagKey.TagKeyString;
+import io.opencensus.tags.TagValue;
+import io.opencensus.tags.TagValue.TagValueString;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import javax.annotation.Nullable;
+
+/**
+ * Util methods to convert OpenCensus Stats data models to StackDriver monitoring data models.
+ */
+final class StackdriverExportUtils {
+
+  // TODO(songya): do we want these constants to be customizable?
+  @VisibleForTesting static final String LABEL_DESCRIPTION = "OpenCensus TagKey";
+
+  // Construct a MetricDescriptor using a View.
+  @Nullable
+  static MetricDescriptor createMetricDescriptor(View view, String projectId) {
+    if (!(view.getWindow() instanceof Cumulative)) {
+      // TODO(songya): Only Cumulative view will be exported to Stackdriver in v0.1
+      return null;
+    }
+
+    MetricDescriptor.Builder builder = MetricDescriptor.newBuilder();
+    String viewName = view.getName().asString();
+    // Name format refers to
+    // cloud.google.com/monitoring/api/ref_v3/rest/v3/projects.metricDescriptors/create
+    builder.setName(String.format("projects/%s", projectId));
+    builder.setType(createMetricType(viewName));
+    builder.setDescription(view.getDescription());
+    builder.setUnit(view.getMeasure().getUnit());
+    builder.setDisplayName("OpenCensus/" + viewName);
+    for (TagKey tagKey : view.getColumns()) {
+      builder.addLabels(createLabelDescriptor(tagKey));
+    }
+    builder.setMetricKind(createMetricKind(view.getWindow()));
+    builder.setValueType(createValueType(view.getAggregation(), view.getMeasure()));
+    return builder.build();
+  }
+
+  // Construct a MetricDescriptor.Type. From the documentation:
+  // cloud.google.com/monitoring/api/ref_v3/rest/v3/projects.metricDescriptors
+  // For now only support custom metrics.
+  private static String createMetricType(String viewName) {
+    return String.format("custom.googleapis.com/opencensus/%s", viewName);
+  }
+
+  // Construct a LabelDescriptor from a TagKey
+  @VisibleForTesting
+  static LabelDescriptor createLabelDescriptor(TagKey tagKey) {
+    LabelDescriptor.Builder builder = LabelDescriptor.newBuilder();
+    builder.setKey(tagKey.getName());
+    builder.setDescription(LABEL_DESCRIPTION);
+    LabelDescriptor.ValueType valueType = tagKey.match(
+        Functions.returnConstant(ValueType.STRING),
+        Functions.returnConstant(ValueType.INT64),
+        Functions.returnConstant(ValueType.BOOL),
+        Functions.returnConstant(ValueType.UNRECOGNIZED));
+    builder.setValueType(valueType);
+    return builder.build();
+  }
+
+  // Construct a MetricKind from an AggregationWindow
+  @VisibleForTesting
+  static MetricKind createMetricKind(AggregationWindow window) {
+    return window.match(
+        Functions.returnConstant(MetricKind.CUMULATIVE),               // Cumulative
+        // We don't support exporting Interval stats to StackDriver in v0.1
```

Comment:
I would rephrase it as:
// We don't support exporting Interval stats to StackDriver in this version.

Same everywhere. The current version of the library is 0.8.0 I know that this is the first stats release but it is confusing.

Reply:
Done.